pyaedt/modules/solutions.py

```diff
@@ -1071,338 +1075,248 @@ def ifft_to_file(
+                    self._is_array.append(True)
+                    self._component_objects.append(metadata["component_objects"])
+                    self._array_dimension.append(metadata["array_dimension"])
+                    self._cell_position.append(metadata["cell_position"])
+                    self._lattice_vector.append(metadata["lattice_vector"])
+                else:
+                    self._is_array.append(False)
+
+        self.port_index = self._get_port_index()
+        if not self._get_port_index:  # pragma: no cover
+            raise Exception("Wrong port index load.")
+        self.__model_units = "meter"
+        self.frequency = self.frequencies[0]
+
+        self.a_min = 9999999999
+        self.a_max = 0
+        self.b_min = 9999999999
+        self.b_max = 0
+        if self.port_index:
+            for row, col in self.port_index.values():
+                self.a_min = min(self.a_min, row - 1)
+                self.a_max = max(self.a_max, row - 1)
+                self.b_min = min(self.b_min, col - 1)
+                self.b_max = max(self.b_max, col - 1)
 
     @property
     def frequency(self):
-        """Get/set the Active Frequency.
-
-        Returns
-        -------
-        float
-        """
+        """Active frequency."""
         return self._frequency
 
     @frequency.setter
     def frequency(self, val):
         if val in self.frequencies:
-            self._frequency = val
-            self.ffd_dict = self._all_solutions[self.frequencies.index(val)]
-            self._init_ffd()
+            freq_index = self.frequencies.index(val)
+            eep_file_info = self._eep_file_info_list[freq_index]
+            init_flag = self._init_ffd(eep_file_info)
+            if init_flag:
+                self._frequency = val
+                self._freq_index = self.frequencies.index(val)
+                self.eep_file_info = self._eep_file_info_list[self._freq_index]
+                self.farfield_data = self.combine_farfield()
+            else:  # pragma: no cover
+                self.logger.error("Wrong farfield information.")
+        else:  # pragma: no cover
+            self.logger.error("Frequency not available.")
 
     @property
-    def phase_offset(self):
-        """Additional phase offset in degrees on each port. Useful when element has more than one port.
+    def frequency_value(self):
+        """Frequency value in Hz."""
+        if isinstance(self.frequency, str):
+            frequency, units = decompose_variable_value(str(self.frequency))
+            return unit_converter(frequency, "Freq", units, "Hz")
+        else:
+            return float(self.frequency)
 
-        Returns
-        -------
-        list
-        """
+    @property
+    def phase_offset(self):
+        """List of additional phase offset in degrees on each port. Useful when element has more than one port."""
```

Comment:
```suggestion
        """List of additional phase offset in degrees on each port. This property
        is useful when an element has more than one port."""
```
Following a brief description (which is a sentence fragment), use complete sentences to provide additional information.